src/main/java/org/opentripplanner/netex/loader/mapping/TripPatternMapper.java

```diff
@@ -33,82 +33,85 @@
  * <p>
  * Headsigns in NeTEx are only specified once and then valid for each subsequent TimeTabledPassingTime until a new
  * headsign is specified. This is accounted for in the mapper.
+ * <p>
+ * THIS CLASS IS NOT THREADSAFE! This mapper store its intermediate results as part of its state.
  */
 class TripPatternMapper {
```

Comment:
The changes in this class has nothing to do with the Notices - only refactoring so it now return the OTP Model objects - instead of inserting them into the builder passed in. In the previous design it was very hard to find out which entities where inserted where. No it all happens in the root mapper: NetexMapper.